src/OrchardCore.Modules/OrchardCore.ContentFields/OrchardCore.ContentFields.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.ContentManagement\OrchardCore.ContentManagement.csproj" />
```

Comment:
You should only need the abstractions project.

Reply:
In the DefaultContentPickerResultProvider I'm referencing the ContentItemIndex class which is in that project.

Reply:
Correct, I thought it was a reference to the module.